src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -426,6 +426,16 @@ private IQ handleIq(IQ iq)
     {
         IQ responseIq = null;
 
+        EntityBareJid fromJid = iq.getFrom().asEntityBareJidIfPossible();
+        String fromJidStr = fromJid.toString().toLowerCase();
+        if (fromJid == null
+            || !this.config.getMucJids().stream().anyMatch(
```

Comment:
am i understanding it right that this is verifying that the message came from a user in the muc?  does this need to be configurable?

Reply:
Yes, that's correct.

Reply:
Does this mean jicofo has to be configured as MUC_JID as well? Without it, we see
```Received an IQ from a non-MUC member: focus@auth....```
with it, we see
```org.jivesoftware.smackx.muc.MultiUserChatException$NotAMucServiceException: Can not join 'focus@auth....', because 'auth....' does not provide a MUC (XEP-45) service.```

Reply:
You see this in jvb?

Reply:
Yes, in order to get JVB2 running with the configuration from `docker-jitsi-meet`, I followed https://github.com/jitsi/jicoco/pull/54/files. Then I saw
```Received an IQ from a non-MUC member: focus@auth...```
and jitsi-meet (JS) didn't connect.
Adding the focus to the `MUC_JIDS` fixes that, but logs the `NotAMucServiceException`

Reply:
The only known problem that I know is that this in the jvb config cause problem and needs to be adjusted or removed:
org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP=focus@auth.${JVB_HOSTNAME}/.*
Should be adjusted to:
jvbbrewery@internal.auth.example.com/focus

I see that is set in in docker jvb config, I suppose your jvb config looks like:
https://github.com/jitsi/docker-jitsi-meet/blob/master/jvb/rootfs/defaults/sip-communicator.properties

Reply:
Ok, that works now (with just the `AUTHORIZED_SOURCE_REGEXP` and no extra `MUC_JID`). It appears that I was not using the very latest jicofo. Thanks for your help @damencho !